README.md

```diff
@@ -21,12 +21,21 @@
 [![Sandbox badge](https://img.shields.io/badge/Stage-Sandbox-yellow)](https://github.com/eiffel-community/community/blob/master/PROJECT_LIFECYCLE.md#stage-sandbox)
 
 # Eiffel Events SDK – Go
-This repository contains data types, constants, and functions for working with Eiffel events in the [Go](https://golang.org/) language, including serialization to and from JSON. Its goal is to make it easy to create and process Eiffel events in Go.
+This repository contains data types, constants, and functions for working with Eiffel events in the [Go](https://golang.org/) language, including marshaling to and from JSON. Its goal is to make it easy to create and process Eiffel events in Go.
```

Comment:
Yeah :-/ For some reason I started with >80 chars (probably because that's what the template repo uses) but stuck to 80 chars when I added paragraphs of my own. I'll clean this up at some point.